### ISSUE TITLE
Cannot reach OctoFarm using 'Installation Production' Documentation

### ISSUE THREAD

**Issue by adamsiem** [NotExpectedYet/OctoFarm#32]
**It started... woohoo!**

<img width="641" alt="image" src="https://user-images.githubusercontent.com/1068543/79093442-4f3d5100-7d22-11ea-91ca-0140b4c248c8.png">

**But...**

<img width="304" alt="image" src="https://user-images.githubusercontent.com/1068543/79093395-2ddc6500-7d22-11ea-944f-98380cd960cd.png">

I followed the documentation for 'Installation Production > Debian/Linux Specific', but cannot access OctoFarm on my local network at port 4000.

NOTE: This is on Raspbian, and I have Pi-Hole running on the same device, successfully, at the IP address only, i.e. no custom port for the web-server.

**The productionError.log is empty... woohoo!**

GNU nano 3.2 productionError.log

**My config looks... good...?**

GNU nano 3.2             config/db.js

```
module.exports = {
  MongoURI: "mongodb://127.0.0.1:27017/octofarm"
};
//Example Local URL: "mongodb://192.168.1.5:27017/octofarm"
//Example Remote URL: "mongodb+srv://s1mpleman:<YOUR PASSWORD>$
```

This resembles Issue https://github.com/NotExpectedYet/OctoFarm/issues/16 (Cannot reach OctoFarm using Docker)

**Comment by NotExpectedYet** [comment]
PiHole image uses Rasbian so there's no MongoDB support for the version that's required. 

Your config looks fine, but you'd have to use an External MongoDB installation to get it to work. I gather it's not starting because it's not able to connect to the local MongoDB instance. Sorry there isn't more loggin but it usually produces are error for that although I've not specifically tried it. 

You could also setup the 64-bit Ubuntu version on the Pi and install both OctoFarm + PiHole as a dependancy on that if you still wanted to keep your Pi for those.

**Comment by gmccauley** [comment]
+ 1 on ubuntu.  I used an old Pi 3B I had laying around and installed the 64 bit version of ubuntu and it's working great.

**Comment by adamsiem** [comment]
I confirmed this is resolved on Ubuntu Server 64-bit 18.04 LTS on a Raspberry Pi 4 B. Nice work!
